Description: Use VFPv3-D16 FPU for ARM builds
 Jani writes: The D16 part was Debian/Ubuntu specific, IIRC we define hardfloat
 in that particular variant (16 double registers) or we had a different naming
 for some reason.
Author: Jani Monoses <jani@ubuntu.com>
Upstream-Bug: https://ghc.haskell.org/trac/ghc/ticket/5914
Launchpad-Bug: https://ghc.haskell.org/trac/ghc/ticket/5914

Index: ghc/aclocal.m4
===================================================================
--- ghc.orig/aclocal.m4	2016-01-14 16:19:24.858462215 +0100
+++ ghc/aclocal.m4	2016-01-14 16:19:24.854462134 +0100
@@ -420,7 +420,7 @@
                 )],
                 [changequote(, )dnl
                  ARM_ISA=ARMv7
-                 ARM_ISA_EXT="[VFPv3,NEON]"
+                 ARM_ISA_EXT="[VFPv3D16,NEON]"
                  changequote([, ])dnl
                 ])
         ])
